src/telemetry.rs

```diff
@@ -50,35 +45,36 @@ pub fn configure<T>(config: &SubscriberConfig<T>) -> Tracer {
 
     let collector_url = normalize_collector_url(&telemetry.collector_url);
 
-    let otlp_exporter = opentelemetry_otlp::new_exporter()
-        .http()
-        .with_endpoint(collector_url);
+    let otlp_exporter = SpanExporter::builder()
+        .with_http()
+        .with_endpoint(collector_url)
+        .build()
+        .expect("Failed to configure the OpenTelemetry OTLP span exporter");
 
     let resource = Resource::new(vec![
         KeyValue::new("environment", config.env.to_string()),
         KeyValue::new("country", config.country.to_string()),
         KeyValue::new("service.name", telemetry.service_name.clone()),
     ]);
 
-    let tracer_provider = opentelemetry_otlp::new_pipeline()
-        .tracing()
-        .with_exporter(otlp_exporter)
-        .with_trace_config(trace::Config::default().with_resource(resource))
-        .install_batch(runtime)
-        .expect("Failed to configure the OpenTelemetry tracer provider");
+    let tracer_provider = sdktrace::TracerProvider::builder()
+        .with_batch_exporter(otlp_exporter, runtime)
+        .with_config(sdktrace::Config::default().with_resource(resource))
+        .build();
 
     set_tracer_provider(tracer_provider.clone());
 
-    tracer_provider
-        .tracer_builder("prima-tracing")
+    let scope = InstrumentationScope::builder(env!("CARGO_PKG_NAME"))
         .with_version(env!("CARGO_PKG_VERSION"))
-        .build()
+        .build();
+
+    tracer_provider.tracer_with_scope(scope)
```

Comment:
Do we need to create a scope? I think that a scope is supposed to start at an entrypoint(eg. Http call). I think this will create a global scope, encompassing everything that happens to the application

Reply:
AFAIK [this is how they ported it in their PR](https://github.com/open-telemetry/opentelemetry-rust/compare/opentelemetry-otlp-0.26.0...opentelemetry-otlp-0.27.0#diff-e97d52788c4f281d467efd4de6ba6e9241d2ed09434f7dc36dcd6b447950aa38L124) :man_shrugging: 

We may not need to do it at all, but I think this was a "problem" before too